app/src/main/java/com/sunasterisk/travelapp/ui/list/hotel/HotelListActivity.kt

```diff
@@ -0,0 +1,58 @@
+package com.sunasterisk.travelapp.ui.list.hotel
+
+import android.view.MenuItem
+import com.sunasterisk.travelapp.R
+import com.sunasterisk.travelapp.ui.adapter.HotelAdapter
+import com.sunasterisk.travelapp.ui.base.BaseActivity
+import com.sunasterisk.travelapp.ui.custom.TabName
+import com.sunasterisk.travelapp.utils.location
+import com.sunasterisk.travelapp.utils.screenHeight
+import kotlinx.android.synthetic.main.activity_hotel_list.*
+import kotlinx.android.synthetic.main.include_hotel_list_front.*
+
+class HotelListActivity : BaseActivity() {
+
+    private val adapter: HotelAdapter = HotelAdapter()
+
+    override val layoutResource = R.layout.activity_hotel_list
```

Comment:
Ưu tiên thuộc tính, phương thức override viết trước 
```suggestion
   override val layoutResource get() = R.layout.activity_hotel_list
     private val adapter = HotelAdapter()
```

Reply:
ok ông